indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -500,6 +500,11 @@ th.i-table {
             background-color: $dark-gray;
         }
     }
+
+    .table-content {
+        max-height: 50em;
+        overflow: scroll;
```

Comment:
The scroll bars shouldn't be visible when it's not necessary. Try overflow: auto
(sorry for the other comment, wrong place!)